tools/ax_lib_readline.m4

```diff
@@ -211,21 +211,23 @@ AC_DEFUN([AX_LIB_READLINE], [
   	[search for readline in DIR/include and DIR/lib])],
 	[ac_cv_rl_prefix=$with_readline])
 
-  AC_CHECK_HEADERS([readline/readline.h], [
-    AC_CHECK_HEADERS([readline/history.h], [
-      RL_LIB_READLINE_VERSION
+  AS_IF([test "x$with_readline" != xno], [
```

Comment:
Are these changes made manually or did you update it to the newest version of this file?

Reply:
Manually.